src/libraries/System.Net.Security/tests/FunctionalTests/NegotiateStreamStreamToStreamTest.cs

```diff
@@ -192,6 +193,11 @@ public async Task NegotiateStream_StreamToStream_Authentication_EmptyCredentials
         {
             string targetName = "testTargetName";
 
+            if (PlatformDetection.IsWindowsServer2025)
+            {
+                throw new SkipTestException("Empty credentials not supported on Server 2025");
```

Comment:
Wouldn't it be better to do
```c#
[ActiveIssue("https://github.com/dotnet/runtime/issues/please_file_an_issue", typeof(PlatformDetection), nameof(PlatformDetection.IsWindowsServer2025))]
```

Reply:
I would for temporary fix. In this case I don't expect any more work e.g. there is no issue to link to.

Reply:
Is this something we'd want to doc?

Reply:
I don't think so. Nobody really should be using empty credentials anyway. e.g. there _should_ be name and password. The difference is that in the past the handshake would fail but now the credentials are rejected right at the beginning.

Reply:
Do we needt o document a different exception?

Reply:
We do not document exact locations and timing for such errors. It still surfaces as `AuthenticationException` that _should_ be already in the docs. And the empty credentials is really not a normal/valid case. I would feel the answer is no.  If anything, we can enhance the test. But I'm not sure it is worth it.

Reply:
@wfurt - not sure if this is specific to Server 2025.  Saw the same failure on Win11 client (windows.11.amd64.client.open.rt) -- https://helixr1107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-112032-merge-b93d34177f5e4607a1/System.Net.Security.Tests/3/console.084cc044.log?helixlogtype=result.  
You might want to take a deeper look at this.

Reply:
Maybe not - that one was `NegotiateStream_StreamToStream_Authentication_EmptyCredentials_Fails` 😆 - I guess if you don't feel it's a valid case it doesn't warrant too much investigation.

Reply:
we should update the test to be more resilient if it is more prevalent. It is posible that the new behavior may also come to other OSes via Windows update. cc: @rzikm